### DISCUSSION TITLE
Maybe a script for required packages installation is a good way?

### DISCUSSION THREAD

**Discussion by yinghaoyu** [ayamir/nvimdots#1076] [Ideas]
After I turn to an other host, I must install packages one by one. It's killing me.

**Comment by charliie-dev** [comment]
It's already [here](https://github.com/ayamir/nvimdots/wiki/Prerequisites).
What else do u expect?
If u need extra pkgs, why not just write your own script or use a container? it's not hard.

**Reply by yinghaoyu** [reply]
Thanks. I had noticed with `Wiki`, but it not seem a true `script`, especially at an off-line host.

**Reply by yinghaoyu** [reply]
Maybe it should be moved to `README.md`.

**Reply by charliie-dev** [reply]
if your host is offline, then no script can help u. find your own way.

**Comment by ayamir** [comment]
> After I turn to an other host, I must install packages one by one, It's killing me.

A customized docker image is suitable for you. I'm going to write a dockerfile for `nvimdots` when spare.

**Comment by ayamir** [comment]
@yinghaoyu @CharlesChiuGit @Jint-lzxy 
https://hub.docker.com/repository/docker/ayamir/nvimdots/general

![image](https://github.com/ayamir/nvimdots/assets/61657399/0e210ed8-6728-4eb5-82b1-069361e0170b)
